third_party/gpus/cuda_configure.bzl

```diff
@@ -35,6 +35,10 @@ _DEFAULT_CUDA_COMPUTE_CAPABILITIES = ["3.5", "5.2"]
 # BEGIN cc_configure common functions.
 def find_cc(repository_ctx):
   """Find the C++ compiler."""
+  # On Windows, we use Bazel's MSVC CROSSTOOL for GPU build
+  # Return a dummy value for GCC detection here to avoid error
+  if _cpu_value(repository_ctx) == "Windows":
+    return "/usr/bin/dummycompiler"
```

Comment:
Can we make this name like this, so users can understand what is going on if they use the wrong config to compile on windows?

Something like: `/use/--config x64_msvc_windows/instead`